db/db_impl_compaction_flush.cc

```diff
@@ -511,7 +513,8 @@ Status DBImpl::AtomicFlushMemTablesToOutputFiles(
 
 void DBImpl::NotifyOnFlushBegin(ColumnFamilyData* cfd, FileMetaData* file_meta,
                                 const MutableCFOptions& mutable_cf_options,
-                                int job_id, TableProperties prop) {
+                                const Status& st, int job_id,
```

Comment:
I don't think it makes much sense to pass Status into `NotifyOnFlushBegin` given it's always going to be default-constructed (I assume ok?). 

Simpler to revert the change to the method, and just default-construct it at line 539.

Reply:
Defaults to ok. I just copied the OnCompaction**** API which passes a status into OnCompactionBegin, but I did some digging and they have some pre-condition that you have to meet before OnCompactionBegin is called so the status isn't necessarily OK.  Reverted the NotifyOnFlushBegin signature change, manually set the status in the function to OK for clarity.